upa/test/upaTests.ts

```diff
@@ -804,8 +806,9 @@ describe("UPA", async () => {
     });
 
     it("submitMultipleCheckReceipts", async () => {
-      const { upa, s1, s2, s3, s1_tx, s2_tx, s3_tx } =
-        await loadFixture(deployAndSubmit);
+      const { upa, s1, s2, s3, s1_tx, s2_tx, s3_tx } = await loadFixture(
```

Comment:
There are a tonne of formatting changes which inflate the PR and make it harder to review.
Do you know which version of prettier you are using?